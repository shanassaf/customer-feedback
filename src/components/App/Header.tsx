import React from 'react';
import styled from 'styled-components';

const StyledHeader = styled.header``;

export const Header = () => (
	<StyledHeader>
		<img src="../images/checkout.jpeg" alt="Checkout logo" />
	</StyledHeader>
);
