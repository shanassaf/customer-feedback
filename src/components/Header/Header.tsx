import React from 'react';
import styled from 'styled-components';
import Checkout from '../../images/checkout.svg';

const StyledHeader = styled.header`
	padding: 15px;
`;

export const Header = (): JSX.Element => (
	<StyledHeader>
		<img height="30px" src={Checkout} alt="Checkout logo" />
	</StyledHeader>
);
