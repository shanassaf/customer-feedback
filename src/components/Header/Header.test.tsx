import React from 'react';
import { render } from '@testing-library/react';
import { Header } from './Header';

describe('Header', () => {
	test('component renders correctly', () => {
		const { asFragment } = render(<Header />);
		expect(asFragment()).toMatchSnapshot();
	});
});
