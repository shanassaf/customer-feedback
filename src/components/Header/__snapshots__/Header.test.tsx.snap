// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<DocumentFragment>
  <header
    class="sc-bdnylx cdDTBJ"
  >
    <img
      alt="Checkout logo"
      height="20px"
      src="checkout.svg"
    />
  </header>
</DocumentFragment>
`;
